aries-site/src/examples/templates/dashboards/components/ChartCard.js

```diff
@@ -49,6 +49,7 @@ export const ChartCard = ({
               ? { horizontal: 'large', top: body.pad, bottom: 'large' }
               : footerTheme.pad
           }
+          border="top"
```

Comment:
Designs call for `border-weak` as the color

Reply:
will fix

Reply:
Done.